src/CMakeLists.txt

```diff
@@ -57,6 +66,9 @@ set(sources main.cpp qheifhandler.cpp)
 
 add_library(qheif MODULE ${sources})
 
+target_include_directories(qheif PRIVATE ${libheif_INCLUDE_DIRS})
+target_link_directories(qheif PRIVATE ${LIBHEIF_LIBRARY_DIRS})
```

Comment:
a typo, CMake variable names are case sensitive, LIBHEIF_LIBRARY_DIRS is not effect